pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable_test.go

```diff
@@ -170,6 +170,17 @@ func TestIsSchedulableAfterNodeChange(t *testing.T) {
 			},
 			expectedHint: framework.Queue,
 		},
+		{
```

Comment:
Can you add a new test case that when both oldObj and newObj is `Unschedulable: false`, we return QueueSkip. It shows this PR's improvement since we returned Queue to such match -> match scenario .

Reply:
done